spec/design.md

```diff
@@ -174,22 +180,32 @@ The return value is `constructor`, except it is an instance of class
 `Class`, which defines properties that describe the rest of the class.
 
 Example:
+
 ```{R}
-Child <- defineClass("Range", Vector, function(start, end) {
+Range <- defineClass("Range", 
+  Vector, 
+  function(start, end) {
     stopifnot(is.numeric(start), is.numeric(end), end >= start)
-    Range@new(start=start, end=end)
-}, validity = function(object) {
-    if (end < start)
-        "end must be greater than or equal to start"
-}, properties = c(start = "numeric", end = "numeric"))
+    Range@new(start = start, end = end)
+  }, 
+  validity = function(object) {
+    if (end < start) {
+      "end must be greater than or equal to start"
+    }
+  }, 
+  properties = c(start = "numeric", end = "numeric")
+)
+
+Range(start = 1, end = 10)
 ```
 
 ### Properties
 
-Calling `defineProperty()` define a new property. It has the
+Calling `newProperty()` define a new property. It has the
```

Comment:
I think this needs fleshing out — esp whether the name belongs to the property of the property list. Is it:

```R
defineClass("Range", ..., properties = list(newProperty("start", "numeric"))
# OR 
defineClass("Range", ..., properties = list(start = newProperty("numeric"))
```